### DISCUSSION TITLE
Howto on Homeassistant Docker Container

### DISCUSSION THREAD

**Discussion by 555wolfman** [homeassistant-apps/app-cloudflared#395] [Q&A] [answered]
Is there a way to integrate this using Homeassistant Docker Container?

**Comment by brenner-tobias** [comment, marked as answer]
Not via this add-on. I suggest that you use the [Cloudflared Docker container](https://hub.docker.com/r/cloudflare/cloudflared) to run it parallel to your HA docker.
I suggest to use it with a remote configured tunnel and then define the hosts (incl. HA) in the Cloudflared Zero Trust Dashboard.
